crane-cloud/frontend
# fx: change the logic for fetching activity logs

# Description

This PR is to change the base URL for  activity logs to the new microservice URL

## Type of change



## Trello Ticket ID

https://trello.com/c/Gazjtbbq

## How Can This Been Tested?

Run this application and add the environment variable
```REACT_APP_ACTIVITY_LOGS_API_URL``` that should be the base URL to the new app logger activities


## Checklist:
